### PR TITLE
Week 03 Reflection Contribution from mresendiz3-stat697

### PR DESCRIPTION
**Pull request by mariano-resendiz**

### CODE REVIEW

File: week-03-reflections/mresendiz3-stat697.md [hunk 1]
```diff
@@ -1,25 +1,101 @@
 
 # Questions about Problems and Recipes
 
+[Chapter 3, Problem 4]
+* Question (mresendiz3-stat697): Can the inner join syntax compare across data types?
+* Answer: Use inner join with columns that have the same data type. The columns don't necessarily need to have the same name.
```
Comment (prof-lankham) [inline week-03-reflections/mresendiz3-stat697.md]: Answers should also have author attributions

**Review by prof-lankham — CHANGES_REQUESTED** [review]
Please update PR, adding author attribution to all answers, and DM me in Slack with PR URL when ready for re-evaluation.

File: week-03-reflections/mresendiz3-stat697.md (hunk 1 above)
Comment (mariano-resendiz) [reply week-03-reflections/mresendiz3-stat697.md]: Author attributions added; thanks!

**Review by mariano-resendiz — COMMENTED** [review]
Author attribution added.

File: week-03-reflections/mresendiz3-stat697.md (hunk 1 above)
Comment (prof-lankham) [reply week-03-reflections/mresendiz3-stat697.md]: Author attribution means to put your GitHub user name in parentheses after each Question/Answer, which you're currently only doing with Questions.

File: week-03-reflections/mresendiz3-stat697.md [hunk 2]
```diff
@@ -1,25 +1,102 @@
 
 # Questions about Problems and Recipes
 
+[Chapter 3, Problem 4]
+* Question (mresendiz3-stat697): Can the inner join syntax compare across data types?
+* Answer: Use inner join with columns that have the same data type. The columns don't necessarily need to have the same name. 
+* Author attribution: SAS Institute Inc. 2014. SAS� Certification Prep Guide: Advanced Programming for SAS�9, Fourth Edition. Cary, NC: SAS Institute Inc.
```
Comment (prof-lankham) [inline week-03-reflections/mresendiz3-stat697.md]: Citing the textbook is not necessary.

**Review by prof-lankham — CHANGES_REQUESTED** [review]
Please update PR, adding author attribution to all answers, and DM me in Slack with PR URL when ready for re-evaluation.

File: week-03-reflections/mresendiz3-stat697.md (hunk 2 above)
Comment (mariano-resendiz) [reply week-03-reflections/mresendiz3-stat697.md]: I took the citation off and added the author attribution to the W3RC.

**Pull request merged**